wcomponents-core/src/main/java/com/github/bordertech/wcomponents/AbstractTreeTableDataModel.java

```diff
@@ -21,10 +21,8 @@
 	private final TableTreeNode root;
 
 	/**
-	 * <p>
 	 * Creates a SimpleTableDataModel containing the given data. Note that applications may wish to create their own
 	 * TableTreeNode extensions that dynamically load data on e.g. the first call to setExpanded(true)
```

Comment:
Please add full stop to end of this line (it was previously mis-placed on the line you deleted).